src/locales/us/en/index.json

```diff
@@ -75,6 +75,7 @@
   "shouldContainALowercase": "Should contain a lowercase letter",
   "shouldContainASymbol": "Should contain a symbol",
   "showPassword": "Show password",
+  "signalsEvaluation": "Evaluating device information...",
```

Comment:
Since this message is a full sentence, I don't think we need the ellipses. But, if we keep it, let's place a space between the the word and the ellipses.